server/app.js

```diff
@@ -34,7 +34,7 @@ app.use(urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(express.static(join(__dirname, "public")));
 
-app.use("/", indexRouter);
+app.use("/", langingRouter);
```

Comment:
```suggestion
app.use("/api", langingRouter);
```
If the /api is not included, then there will be multiple routes with the same path (Frontend and Backend)

Reply:
included /api to prevent multiple routes with the same path